inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -31,7 +31,6 @@
 
 @hepnames.over('acquisition_source', '^541[10_].')
 @utils.for_each_value
-@utils.filter_values
```

Comment:
Please add extra point in your comment message explaining why we're getting rid of `filter_values` decorator